src/explorer/trialApp/TrialAppTreeItem.ts

```diff
@@ -95,6 +98,13 @@ export class TrialAppTreeItem extends SiteTreeItemBase implements ISiteTreeItem
     public isAncestorOfImpl?(contextValue: string | RegExp): boolean {
         return contextValue === TrialAppTreeItem.contextValue;
     }
+
+    public compareChildrenImpl(item1: AzExtTreeItem, item2: AzExtTreeItem): number {
```

Comment:
I'd feel more comfortable if you checked item1 as well.  The `super.compareChildrenImpl` sorts by alphabetical order, so if item1 is the `GenericTreeItem`, it'll most likely be sorted underneath whatever other node you have since it starts with 's'.